### PR TITLE
Problem: unsafe experimental flag is not needed 

### PR DESCRIPTION
**Pull request by thomas-nguy**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

Remove unsafe experimental flag

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Review by mmsqe — COMMENTED** [review]
do we still keep package [experimental](https://github.com/crypto-org-chain/cronos/blob/d3e567f49b3ba9d8aa93f426d1ff799e79de2652/cmd/cronosd/experimental/util.go#L1) and seems still have experimental in [test_helpers](https://github.com/crypto-org-chain/cronos/blob/d3e567f49b3ba9d8aa93f426d1ff799e79de2652/app/test_helpers.go)

File: integration_tests/test_upgrade.py [hunk 1]
```diff
@@ -97,19 +97,18 @@ def test_cosmovisor_upgrade(custom_cronos: Cronos):
...
 
     # estimateGas for an erc20 transfer tx
     old_gas = contract.functions.transfer(ADDRS["community"], 100).build_transaction(
         {"from": ADDRS["validator"]}
     )["gas"]
 
-    plan_name = "v1.0.0"
-    rsp = cli.gov_propose_v0_7(
+    plan_name = "v2.0.0"
+    rsp = cli.gov_propose_legacy(
```
Comment (thomas-nguy) [inline integration_tests/test_upgrade.py:111]: the new cmd required a json file, I fallback to the legacy

File: app/upgrades.go [hunk 2]
```diff
@@ -1,78 +1,24 @@
...
-		params := app.EvmKeeper.GetParams(ctx)
-		params.ExtraEIPs = []int64{}
-
-		// fix the incorrect value on testnet parameters
-		zero := sdkmath.ZeroInt()
-		params.ChainConfig.LondonBlock = &zero
-
-		app.EvmKeeper.SetParams(ctx, params)
+		gravParams := app.GravityKeeper.GetParams(ctx)
+		gravParams.GravityId = "cronos_gravity_testnet"
```
Comment (calvinaco) [inline app/upgrades.go:16]: Should we remove the `_testne` suffix or make the upgrade plan name to `v2.0.0-testnet3`?

**Review by calvinaco — COMMENTED** [review]
LGTM, left a comment on the upgrade handler

File: docs/architecture/adr-001.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 * 12-10-2021: Initial Draft
 * 13-10-2021: Code deletion instead of conditional compilation
 * 16-11-2021: Experimental activation flag
+* 02-02-2022: Removal of experimental activation flag
```
Comment (calvinaco) [inline docs/architecture/adr-001.md:7]:
```suggestion
* 02-02-2023: Removal of experimental activation flag
```

File: app/upgrades.go (hunk 2 above)
Comment (thomas-nguy) [reply app/upgrades.go]: reverted the change in the upgrade handler because its breaking the integration test for some reason. Will do it in a separate PR

**Review by mmsqe — APPROVED** [review]

**Review by yihuang — APPROVED** [review]

**Pull request merged**